.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: java
+jdk:
+  - openjdk6
+  - openjdk7
```

Comment:
We can start with removing two things now and make new branch which targets two JDKs sometimes.
Travis CI will test all PRs and branches, so it would be fine.